src/document/json/rga_tree_split.ts

```diff
@@ -586,7 +630,58 @@ export class RGATreeSplit<T extends RGATreeSplitValue> {
       this.treeByIndex.splayNode(node);
     }
 
-    return [changes, createdAtMapByActor];
+    return [changes, createdAtMapByActor, removedNodeMap];
+  }
+
+  public getRemovedNodesLen(): number {
+    return this.removedNodeMap.size;
+  }
+
+  public getCleanupRemovedNodes(ticket: TimeTicket): number {
```

Comment:
In my view,`cleanupRemovedNodes` function's purpose is to purge and return removed node length. so when i just look into the the function name(`cleanupRemovedNodes`) at first, it just feels like cleaning node, not return anything. So I chose the name to `getCleanupRemovedNodes ` 🤔

https://github.com/kettanaito/naming-cheatsheet#context

How about separating the function for purge and getting removed node?

Reply:
I think `cleanupRemovedNodes` is more appropriate because the main logic of this function is cleanup and it's not strange that the count of removed nodes, which is additional info of this logic is returned. I expect there will be many similar examples where additional information will be returned to a method of type delete elements of a collection.

https://docs.oracle.com/javase/8/docs/api/java/util/ArrayList.html#removeIf-java.util.function.Predicate-